🎉  Add end of project event (#33)
UNIVERSE-HPC/UNIVERSE-HPC.github.io: _pages/events/2025-03-27_project-end.md

---
layout: single
classes: wide
title: HPC Ready - Developing RSE and HPC skills
 
permalink: /events/2025-03-27_project-end/
author_profile: false

---

> **Join us for "HPC Ready: Developing RSE and HPC skills" on Thursday 27th March 2025 from 10:00 to 16:00 UTC**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.

Reviewer: Just a thought, the time makes it obvious but we could perhaps highlight that it's a one-day workshop since I feel that tends to engage people somewhat? e.g. 
```suggestion
**Join us for a one-day workshop - "HPC Ready: Developing RSE and HPC skills" - on Thursday 27th March 2025 from 10:00 to 16:00 (GMT/UTC)**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.
```

Everyone is welcome!

> ### [Register now (form to be updated)](https://forms.office.com/Pages/ResponsePage.aspx?id=B3WJK4zudUWDC0-CZ8PTB5xW6C9sTFJAj9JEr0thFURUOTlJQ1RDUDlZUkxNTDJBMVdBSzRDOUlDQi4u)

Reviewer 2: ```suggestion
### [Register now](https://forms.office.com/Pages/ResponsePage.aspx?id=B3WJK4zudUWDC0-CZ8PTB5xW6C9sTFJAj9JEr0thFURUOTlJQ1RDUDlZUkxNTDJBMVdBSzRDOUlDQi4u)
```

Reviewer: You could also potentially use the short link for the form here, although I guess it doesn't actually matter?! 🙂

## Overview

- 🕒 **Thursday 27th March** 2025, **[10:00 - 16:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready%3A+Developing+RSE+and+HPC+skills&iso=20250327T10&p1=1440&ah=6)**

> - 🌍 In person at the University of Edinburgh, Bayes Centre and Online via Zoom

Reviewer 2: ```suggestion
- 🌍 **In person** at the University of Edinburgh, Bayes Centre and **Online** via Zoom
```

> UNIVERSE-HPC is organising an event to mark the end of the three years of the UNIVERSE-HPC project. We'll have talks from members of the project as well as the wider RSE and HPC training community. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in this area! :sparkles:

Reviewer 2: ```suggestion
After three years, the UNIVERSE-HPC project is coming to a close and we will be celebrating this milestone with an event about training for RSEs, with an emphasis on HPC. Members of the wider RSE and HPC training communities will give talks on their work in this field and UNIVERSE-HPC members will share some of our achievements over the course of the project. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in the area of RSE training! ✨ 
```

Reviewer: I don't think I can edit the suggested change above so just noting that in the edited version, the first line has a typo where it says "...we are want to....". You could perhaps say: "_After three years, the UNIVERSE-HPC project is coming to a close and we will be celebrating this milestone with an event..._".

Also, maybe expand HPC where it says with an emphasis on, i.e.: "_with an emphasis on High Performance Computing (HPC)._"?

Tea, coffee, cake and lunch will be provided!

Please read [our event privacy notice](https://www.imperial.ac.uk/media/imperial-college/administration-and-support-services/secretariat/public/ICL---Events-privacy-notice---10-October-2018.pdf).

### Provisional agenda

> - 10:00 - 10:30 Coffees and Meet and Greet ☕

Reviewer 2: ```suggestion
- 10:00 - 10:30 Meet and Greet ☕
```

- 10:30 - 11:15 Introduction and Opening Keynote
- 11:15 - 11:30 Coffee break ☕

> - 11:30 - 12:15 Introduction to learning pathways and invited Ligthning talks

Reviewer 2: ```suggestion
- 11:30 - 12:15 Session on learning pathways
```

> - 12:15 - 13:00 Learning pathway activities (including reporting)

Reviewer 2: ```suggestion
- 12:15 - 13:00 Break-out rooms: Learning pathway activities
```

- 13:00 - 14:00 Lunch break 🍱🥪
- 14:00 - 14:45 Lightning talks
- 14:45 - 15:00 Coffee break ☕

> - 15:00 -15:45 Closing Keynote

Reviewer 2: ```suggestion
- 15:00 - 15:45 Closing Keynote
```

> - 15:45 - 16:00 Closing remarks & evaluation survey

Reviewer 2: ```suggestion
- 15:45 - 16:00 Closing remarks
```

---
layout: single
classes: wide
title: HPC Ready - Three years of developing RSE skills with UNIVERSE-HPC (tentative)
permalink: /events/2025-03-27_project-end/
author_profile: false

---

**Join us for "HPC Ready - Three years of developing RSE skills with UNIVERSE-HPC" on Thursday 27th March 2025 from 10:00 to 16:00 UTC**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.

Everyone is welcome!

### [Register now (form to be updated)](https://forms.office.com/Pages/ResponsePage.aspx?id=B3WJK4zudUWDC0-CZ8PTB5xW6C9sTFJAj9JEr0thFURUOTlJQ1RDUDlZUkxNTDJBMVdBSzRDOUlDQi4u)

## Overview

> - 🕒 **Thursday 27th February** 2025, **[10:00 - 16:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready+-+Three+years+of+developing+RSE+skills+with+UNIVERSE-HPC+%28tentative%29&iso=20250327T10&p1=1440&ah=6)**

Reviewer: I think this should say March 🙂
